platform/tezos/client.go

```diff
@@ -14,7 +14,7 @@ type Client struct {
 func (c *Client) GetTxsOfAddress(address string) ([]Tx, error) {
 	var account Op
 	path := fmt.Sprintf("account/%s/op", address)
-	err := c.Get(&account, path, url.Values{"limit": {"1000"}, "offset": {"0"}})
+	err := c.Get(&account, path, url.Values{"limit": {"5000"}, "offset": {"0"}})
```

Comment:
Why?

Reply:
Yehh, why not just query as much as possible and limit to TxPage size ?

Reply:
For Tezos, some operations are not parsed because is not a transaction, we can miss, but I going to revert, @ArtemGoryunov made this same question

Reply:
Make sense we use 5k, but the Tezos infra it's not good now, we can have some timeouts

Reply:
How much returned by default?